### ISSUE TITLE
On Chrome ver 105.0.5195.54, Account Picture seems to be "escaping" it's intended area in the client panel.

### ISSUE THREAD

**Issue by sirnanos** [pterodactyl/panel#4352]
### Current Behavior

In the client menu, the avatar isn't flush with the UI and seems to be "escaping" on Chrome version 105.0.5195.54. Rebooting the server and browser, incognito, etc doesn't seem to fix this. Switching browsers (eg Safari) fixes this issue.
![image](https://user-images.githubusercontent.com/83611399/188119348-05285db2-3b36-4673-b86a-a6775044704f.png)

### Expected Behavior

In the client menu, the avatar is supposed to be flush with the UI.

### Steps to Reproduce

Update Chrome to 105.0.5195.54 and open the main menu.

### Panel Version

1.10.1

### Wings Version

1.7.0

### Error Logs

```bash
This issue doesn't require error logs, but I'll send them anyway.
Wings: https://ptero.co/ihonilybyv
Panel: https://bin.ptdl.co/2uzuw
```

### Is there an existing issue for this?

- [X] I have searched the existing issues before opening this issue.
- [X] I have provided all relevant details, including the specific game and Docker images I am using if this issue is related to running a server.
- [X] I have checked in the Discord server and believe this is a bug with the software, and not a configuration issue with my specific system.

**Comment by djohts** [comment]
Seems to be fixed on 105.0.5195.102
![image](https://user-images.githubusercontent.com/68508885/188262029-f326e5d0-c502-4bb5-a56b-3e98ed105431.png)
![image](https://user-images.githubusercontent.com/68508885/188262033-163aed42-529a-49cb-94fc-131f6b57af2e.png)

**Comment by sirnanos** [comment]
Fixed on 105.0.5195.102 as @itzJOHv and @Ferks-FK pointed out.
